packages/pluggableWidgets/datagrid-web/src/Datagrid.tsx

```diff
@@ -72,6 +72,13 @@ export default function Datagrid(props: DatagridContainerProps): ReactElement {
                 },
                 [props.columns]
             )}
+            valueForSort={useCallback(
```

Comment:
can't we merge value for sort and for filter? Feels a bit sub-optimal to retrieve the same value twice

Reply:
Done